src/screens/Search/Lists/index.js

```diff
@@ -1,9 +1,6 @@
 import ResultsList from './ResultsList';
+import AlgoResultsList from './AlgoResultsList';
```

Comment:
it doesn't help yet because we're not doing any testing but it's a simpler syntax to understand since everything will always be named there's no question about what you're getting when you import.

if we ever move to TypeScript (I really hope we do post-launch) it will help a lot with importing interfaces. 

honestly, as soon as you stop using `default` exports, you'll realize how much better named exports are